sdk/javascript/src/reporter.ts

```diff
@@ -40,7 +45,12 @@ export function fancyParameterEncoder(paramType: string): [string, (any) => any]
   // Decimals are encoded as uints with 18 decimals of precision on-chain.
   if (paramType === 'decimal') {
     actualParamType = 'uint256';
-    actualParamEnc = (x) => web3.utils.toBN('1000000000000000000').muln(x).toString();
+    actualParamEnc = (x) => web3.utils.toBN(1e6).muln(x).toString();
+  }
+
+  if (paramType == 'symbol') {
+    actualParamType = 'string';
+    actualParamEnc = (x) => x.toUpperCase();
```

Comment:
upppppppppercase??????

Reply:
You mean which do we choose? Rob suggested upper

Reply:
yeah just wondering why since upper case letter are so DiFfiCulT to write.

Reply:
I guess it makes sense since tickers are usually upper 🤷‍♂

Reply:
hm, okay, do it!